### DISCUSSION TITLE
MPI error with few cells per core

### DISCUSSION THREAD

**Discussion by jagmohas** [Alpine-DAV/ascent#1197] [Q&A]
I am using ascent with MPI and found that with few cells per core (for my test it was 30k per core) I get the following error

Fatal error in MPIDI_Cray_shared_mem_coll_tree_reduce: Other MPI error, error stack:
MPIDI_Cray_shared_mem_coll_tree_reduce(202): message sizes do not match across processes in the collective routine: Received 12 but expected 4

However, when I reduced the number of cores to have cells > 50k, it works. This is an AMReX based code. Has anyone experienced such issue. 

Thank you

**Comment by cyrush** [comment]
@jagmohas  Can you share your dataset with us?

You can use an hdf5 extract to create files, here is an example of how to ask for this with Ascent:

https://ascent.readthedocs.io/en/latest/Utilities.html#how-do-i-get-blueprint-files
